src/sentry/plugins/sentry_mail/models.py

```diff
@@ -183,10 +183,11 @@ def get_send_to(self, project, event=None):
 
         return send_to_list
 
-    def add_unsubscribe_link(self, context, user_id, project):
+    def add_unsubscribe_link(self, context, user_id, project, source=None):
```

Comment:
Ah- gotcha I can add to the digest as well. In case we want to distinguish in the analysis i'll give it a different name - let me know if you were going for something else @adhiraj

Reply:
I'd also make this non-optional, there's no reason to make it optional as all unsubscribe links should be tracked